.travis.yml

```diff
@@ -13,9 +13,6 @@ matrix:
         - os: linux
           python: 3.2
           env: TOXENV=py32 VENVVER="<14.0.0" PIPVER="<8.0.0" STVER="<30.0.0"
```

Comment:
Drop Python 3.2 as well.

Reply:
As being said here: https://github.com/DataMedSci/mcpartools/pull/100#discussion_r154989417

We still have (unfortunately) some users running Debian 7 which has Python 3.2 installed. I'd delay dropping python3.2 until May 2018 when Debian 7 will be EOL.
I'll do it gladly !